maintainers/maintainer-list.nix

```diff
@@ -13538,6 +13538,12 @@
     githubId = 3105057;
     name = "Jan Beinke";
   };
+  thenonameguy = {
+    email = "thenonameguy24@gmail.com";
+    name = "Krisztian Szabo";
+    github = "thenonameguy";
+    githubId = 2217181;
+  };
```

Comment:
This change needs to be in a separated commit. 
Besides that, LGTM

Reply:
Thanks for the review! Addressing...

Reply:
Ah I guess the maintainer list additon commit has to come first. Fixing.